pkg/commands/clusterbuilder/create_test.go

```diff
@@ -173,7 +173,7 @@ func testClusterBuilderCreateCommand(t *testing.T, when spec.G, it spec.S) {
 				"--order", "./testdata/order.yaml",
 			},
 			ExpectErr: true,
-			ExpectedOutput: `Error: failed to get canonical service account: configmaps "kp-config" not found
+			ExpectedOutput: `Error: configmaps "kp-config" not found
```

Comment:
Should our output be more about needing to set necessary `kp config` fields?

Reply:
at the moment i'm just returning the error from k8s, what do you think this message should look like

Reply:
Maybe the kp config provider can say something like on not found

`canonical service account is unset, set with "kp config"`